### PR TITLE
[19.09] Specify a default tool error reports plugin config rather than reading it from the sample

### PR DESCRIPTION
**Pull request by natefoo**

### CODE REVIEW

File: lib/galaxy/util/plugin_config.py [hunk 1]
```diff
@@ -8,6 +9,9 @@
 from galaxy.util.submodules import import_submodules
 
 
+PluginConfigSource = collections.namedtuple('PluginConfigSource', ['type', 'source'])
```
Comment (jmchilton) [inline lib/galaxy/util/plugin_config.py:12]: 🎩

**Review by jmchilton — APPROVED** [review]

File: lib/galaxy/util/plugin_config.py (hunk 1 above)
Comment (natefoo) [reply lib/galaxy/util/plugin_config.py:12]: I strongly dislike accessing non-local tuples by index.

Comment (jmchilton) [reply lib/galaxy/util/plugin_config.py:12]: It was a good tophat 🤣

Comment (nsoranzo) [reply lib/galaxy/util/plugin_config.py:12]: Shouldn't also `load_plugins()` been modified at line 40 to use the keys?

Comment (natefoo) [reply lib/galaxy/util/plugin_config.py:12]: @nsoranzo There are other callers still passing a tuple apparently.

Comment (nsoranzo) [reply lib/galaxy/util/plugin_config.py:12]: Are there? I couldn't find any in `lib/galaxy/job_metrics/__init__.py` , `lib/galaxy/tool_util/deps/__init__.py` , `lib/galaxy/tool_util/deps/containers.py` or `lib/galaxy/tools/error_reports/__init__.py` .

Comment (natefoo) [reply lib/galaxy/util/plugin_config.py:12]: [Here's one](https://github.com/galaxyproject/galaxy/blob/5c1380fff60a73261d37ce3a9cb53e5ede56416c/lib/galaxy/tool_util/deps/__init__.py#L126).

Comment (jmchilton) [reply lib/galaxy/util/plugin_config.py:12]: This is going to conflict https://github.com/galaxyproject/galaxy/pull/8616/files isn't it, I shouldn't have taken so long to respond to the review of it.

Comment (jmchilton) [reply lib/galaxy/util/plugin_config.py:12]: https://github.com/galaxyproject/galaxy/pull/8616/commits/28b423b78b8050f9f26d4591a22365736f4ec61f

**Pull request merged**